src/i18n.ts

```diff
@@ -0,0 +1,41 @@
+import i18n from "i18next";
+import { initReactI18next } from "react-i18next";
+
+import Backend from "i18next-http-backend";
+import LanguageDetector from "i18next-browser-languagedetector";
+import { format as date_fnz_tz_format } from "date-fns-tz";
+import enCA from "date-fns/locale/en-CA";
+import frCA from "date-fns/locale/fr-CA";
+// don't want to use this?
+// have a look at the Quick start guide
+// for passing in lng and translations on init
+
+i18n
+  // load translation using http -> see /public/locales (i.e. https://github.com/i18next/react-i18next/tree/master/example/react/public/locales)
+  // learn more: https://github.com/i18next/i18next-http-backend
+  .use(Backend)
```

Comment:
Oh, I see what you were doing - you were setting up an ajax call on our own web server. You don't need to do that, I would just include them directly... they are so small I don't see much benefit in lazy loading them.